docs/developer-guide/create-backend-add-on.md

```diff
@@ -0,0 +1,150 @@
+---
+myst:
+  html_meta:
+    "description": "Create a backend add-on for Plone"
+    "property=og:description": "Create a backend add-on for Plone"
+    "property=og:title": "Create a backend add-on for Plone"
+    "keywords": "Plone, create, add-on, plonecli, mr.bob, bobtemplates.plone, Cookieplone, buildout, backend"
+---
+
+(create-backend-add-on-label)=
+
+# Create backend add-on
+
+This chapter describes how to create a backend add-on for Plone using {term}`plonecli`.
+
+```{note}
+This chapter does not apply for either of the following methods, because they create a backend add-on, as well as install Plone for development.
+
+-   {doc}`/install/create-project-cookieplone`
+-   `uvx cookieplone backend_addon` command
+
+Additionally, if you created a frontend add-on only project using the command `uvx cookieplone frontend_addon` and later realized you need a backend add-on to complement it, then you can create a new project using the command `uvx cookieplone` to generate both the frontend and backend add-ons, then finally move files from your existing project to the new one.
+```
+
+
+## Prerequisites
+
+-   You've installed Plone for development through either of the following methods.
+
+    -   {doc}`/admin-guide/install-buildout`
+    -   {doc}`/admin-guide/install-pip`
```

Comment:
```suggestion
```

we dont need to have Plone installed for creating a backend addon (it is an optional).
```
plonecli create addon src/collective.myaddon
```
creates a backend addon, changing the directory and running:
```
plonecli build
```
builds the addon which can be run and previewed using 
```
plonecli serve
```
runs the the plone server using classic ui from the Plone Version defined during creation of addon or the .mrbob global config

additionally, we can integrate the same addon using `local.cfg` file in buildouts

Reply:
@rohnsha0 does your comment and suggestion still apply? Please reply or resolve this comment. Thank you!